src/status_im2/contexts/profile/settings/header/view.cljs

```diff
@@ -34,13 +37,20 @@
         :profile-picture     profile-picture}]
       [rn/view {:style {:margin-bottom 4}}
        [quo/dropdown
-        {:background :blur
-         :size       :size-32
-         :type       :outline
-         :icon?      true
-         :icon-name  :i/online
-         :on-press   not-implemented/alert}
-        (i18n/label :t/online)]]]
+        {:background     :blur
+         :size           :size-32
+         :type           :outline
+         :icon?          true
+         :no-icon-color? true
+         :icon-name      (:icon (header.utils/visibility-status-type-data status))
+         :on-press       #(rf/dispatch [:show-bottom-sheet
+                                        {:shell?  true
+                                         :theme   :dark
+                                         :content (fn [] [visibility-sheet/view
+                                                          {:customization-color customization-color
+                                                           :status-type         (:status-type
+                                                                                 status)}])}])}
+        (:title (header.utils/visibility-status-type-data status))]]]
```

Comment:
The function `(header.utils/visibility-status-type-data status)` can be moved to `let` var, and we can de-structure the `icon` and `title` from there.